tests/bin/MATCH

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# Source spread-funcs.sh which, at runtime, contains the real definition of
+# MATCH and execute it.
+
+# shellcheck source=tests/lib/spread-funcs.sh 
+. "$TESTSLIB/spread-funcs.sh"
+
+MATCH "$@"
```

Comment:
just for your information there is a project https://github.com/snapcore/snapd-testing-tools where we are moving the tools that could be reused by other projects. There you will find most of the tools that currently are in snapd repo.
You could import those into this project so yo don't need to re-generate them here

Reply:
I am now reusing from that repository, it is a subtree under tests/lib/external